content/articles/deserialization-in-c#/index.md

```diff
@@ -0,0 +1,163 @@
+### Introduction
+We ought to at first see what is serialization in c#. Is the most common way of bringing an object into a structure that tends to be composed in memory.

```

Comment:
Could you avoid repetitions in sentences such as `common way`.

Reply:
Hi @WanjaMIKE , I have made the changes